src/cli.js

```diff
@@ -137,6 +138,8 @@ async function runCmd (argv, stdout, stderr) {
       "-q": "--quiet",
       "--watch": Boolean,
       "-w": "--watch",
+      "--license": Boolean,
+      "-l": "--license",
```

Comment:
Thanks for the PR! 🎉 

Let's change `--license` to be a string so that the use can configure the the path to the output license file.

For example `ncc --license LICENSE.txt`.

Then you can pass this path to `outputFilename`

Reply:
It is relative to the dist directory.

Sounds like a good change, I'll make this update today! Thanks for the review!